priv/templates/nimble_template/lib/otp_app/release_tasks.ex.eex

```diff
@@ -29,4 +28,18 @@ defmodule <%= base_module %>.ReleaseTasks do
 
     Application.load(@app)
   end
+  
+  defp migrate_schemas do
+    for repo <- repos() do
+      {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :up, all: true))
+    end
+  end
+
+  defp migrate_data do
+    for repo <- repos() do
+      source = Ecto.Migrator.migrations_path(repo, "data_migrations")
+
+      {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, source, :up, all: true))
```

Comment:
Small question, can we pass a list as the sources for migrations? 

Then we could just have both the base `migrations path` and `data_migrations path` in a list like `&Ecto.Migrator.run(&1, [base_source, data_source], ...)`

and avoid [the issue](https://nimble-co.slack.com/archives/C0213CHBEPP/p1629191907014800?thread_ts=1629191470.013600&cid=C0213CHBEPP) that we previously had with the order of the migrations 🤔 .

Reply:
Hi, @bterone according to the Ecto.Migrator codebase (https://github.com/elixir-ecto/ecto_sql/blob/v3.7.1/lib/ecto/migrator.ex#L458) it seems like doable but we haven't tested that on Production 😂 let me check that a bit, thanks for the suggestion 👍

Reply:
Hi @bterone I can confirm it is working, I will adjust the template :)

![image](https://user-images.githubusercontent.com/11751745/148047756-fe4893e5-6a30-49fc-9225-57a1b1400e48.png)

Reply:
Update it in https://github.com/nimblehq/elixir-templates/pull/148/commits/eb672a75f78ed4163e42be6bef770de061715f2b @bterone